### ISSUE TITLE
[bug]: "The component was not found. It may not exist at the registry. Please make sure it is a valid component."

### ISSUE THREAD

**Issue by RealMenness** [shadcn-ui/ui#5968]
### Describe the bug

I was putting my v0 code in to cursor AI and it gave me this error after doing the whole procedure thing. I have searched for how to fix this, but all of them are not helping. Please if someone could give me a few minutes of their time, I would be really thankfull. 
![image](https://github.com/user-attachments/assets/6195cb76-8306-49b0-a6f4-4cf642865a6c)

### Affected component/components

styles

### How to reproduce

npx shadcn@latest and add the v0 codebase

### System Info

```bash
windows 10? idk what to put here
```

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues

**Comment by Trung-Nam** [comment]
Same issue and i can't go to https://ui.shadcn.com/

**Comment by navneetguptacse** [comment]
> Same issue when i try to add a component and i can't go to https://ui.shadcn.com/ website

Yes, this issue persists every time I attempt to execute the command for adding any of component. It appears that the problem arises consistently with each and every component.

**Comment by grv24** [comment]
same issue

**Comment by Trung-Nam** [comment]
> > Same issue when i try to add a component and i can't go to https://ui.shadcn.com/ website
> 
> Yes, this issue persists every time I attempt to execute the command for adding any of component. It appears that the problem arises consistently with each and every component.

Oh! After try a lot of ways so i saw that some location hardly to access this website or fail when add more a components. Then try to use vpn then i was fix this

**Comment by shadcn** [comment]
This issue has been automatically marked as stale due to one year of inactivity. It will be closed in 7 days unless there’s further input. If you believe this issue is still relevant, please leave a comment or provide updated details. Thank you. (This is an automated message)

**Comment by shadcn** [comment]
This issue has been automatically closed due to one year of inactivity. If you’re still experiencing a similar problem or have additional details to share, please open a new issue following our current issue template. Your updated report helps us investigate and address concerns more efficiently. Thank you for your understanding! (This is an automated message)
